sdk/core/core-tracing/src/createSpan.ts

```diff
@@ -63,7 +78,12 @@ export function createSpanFunction(args: CreateSpanFunctionArgs) {
     };
 
     const spanName = args.packagePrefix ? `${args.packagePrefix}.${operationName}` : operationName;
-    const span = tracer.startSpan(spanName, spanOptions, tracingOptions.tracingContext);
+
+    let span = tracer.startSpan(spanName, spanOptions, tracingOptions.tracingContext);
+
+    if (isTracingDisabled()) {
+      span = trace.wrapSpanContext(INVALID_SPAN_CONTEXT);
```

Comment:
No need for NoOpSpan and NoOpTracer anymore 😄 This is the recommended approach from OTel

Reply:
Shouldn't we be doing this instead of calling startSpan, when tracing is disabled?